fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/client/fluid/FluidVariantRendering.java

```diff
@@ -50,6 +54,22 @@
 	private static final ApiProviderMap<Fluid, FluidVariantRenderHandler> HANDLERS = ApiProviderMap.create();
 	private static final FluidVariantRenderHandler DEFAULT_HANDLER = new FluidVariantRenderHandler() { };
 
+	static {
+		// Add colored names for vanilla fluids!
+		register(Fluids.WATER, new FluidVariantRenderHandler() {
+			@Override
+			public Text getName(FluidVariant fluidVariant) {
+				return ((MutableText) FluidVariantRenderHandler.super.getName(fluidVariant)).setStyle(Style.EMPTY.withColor(Formatting.BLUE));
+			}
+		});
+		register(Fluids.LAVA, new FluidVariantRenderHandler() {
+			@Override
+			public Text getName(FluidVariant fluidVariant) {
+				return ((MutableText) FluidVariantRenderHandler.super.getName(fluidVariant)).setStyle(Style.EMPTY.withColor(Formatting.RED));
+			}
+		});
```

Comment:
I'm not really sure if this is fitting for Fabric API, but seems like a nice little eyecandy feature.

Reply:
Two mods attempting to do it would lead to a crash for duplicate registrations, unless they catch the resulting exception. Imo it's nicer to have fabric API do it.

Reply:
Personally I would want a way to not apply the colors, maybe another method would fit for the colored one, but this is just out-of-scope in my opinion

Reply:
How so? MI's fluids are all colored, and I am PR'ing this to avoid MI setting the fluid variant render handler of vanilla fluids.

Note that LBA used to color water and lava fluid keys too (in fact that's where we got the idea to color MI fluids from).

Reply:
I like the colors, but I understand the argument too. Hmmm.
If you want to serve both worlds, offer an "enableColoredVanillaFluidNames()", and if any mod calls it -> colored names!

Reply:
What if water is purple and lava is green in my texture pack?

Reply:
Will be a config option in MI.